AudioKit/Common/MIDI/AKMIDIEvent.swift

```diff
@@ -422,4 +415,32 @@ public struct AKMIDIEvent {
         length = newLength
         internalData = Array(internalData[0..<length])
     }
+
+    static func appendIncomingSysex(packet: MIDIPacket) -> AKMIDIEvent? {
+        if let midiBytes = AKMIDIEvent.decode(packet: packet) {
+            AudioKit.midi.incomingSysex += midiBytes
+            if midiBytes.contains(247) {
```

Comment:
What's 247?  Could we use a let constant for clarity?

Reply:
Its the standard 'End of Sysex' MIDI message.

Yes a constant would probably make a lot of sense here - I'll add that now.

Reply:
Ok I see, you have to manually search for an end byte.

Reply:
Yea it is not ideal and you do run into an issue where once you are receiving sysex, you don't stop reading data as sysex until you get that byte. As far as I know - that is MIDI standard.

This is a first iteration and I do welcome any improvements anyone would like to make. I'm just glad we finally support larger sysex messages.